extensions/icd/actions/tfu/__tests__/icd.test.ts

```diff
@@ -56,7 +54,27 @@ describe('Sending a correct input for icd', () => {
     await icd.onActivityCreated(
       generateTestPayload({
         fields: {
-          icd_codes: '[" "]',
+          icd_codes: '[""]',
+        },
+        settings: {},
+      }),
+      onComplete,
+      jest.fn()
+    )
+    expect(onComplete).toHaveBeenCalledWith({
+      data_points: {
+        codes: JSON.stringify([]),
```

Comment:
let's be more clear about what string will be returned now that it's not going to be an array